tests/unit/tf/inputs/test_embedding.py

```diff
@@ -214,6 +214,50 @@ def test_from_pretrained(self, trainable, music_streaming_data: Dataset):
         else:
             np.testing.assert_array_almost_equal(weights, embedding_table.table.embeddings)
 
+    @pytest.mark.parametrize("trainable", [True, False])
+    def test_pretrained_from_InputBlockV2(self, trainable, music_streaming_data: Dataset):
```

Comment:
the `TestEmbeddingTable` class is intended to be more narrowly focused on tests relating to the `EmbeddingTable`. 

Please un-indent this test by 4 spaces so that it's a top-level test in this file.